src/io/observe.hh

```diff
@@ -29,10 +55,14 @@ public:
     DECLARE_INPUT_EXCEPTION(ExcNoInitialPoint,
             << "Failed to find the element containing the initial observe point.\n");
     TYPEDEF_ERR_INFO(EI_RegionName, std::string);
-    TYPEDEF_ERR_INFO(EI_NLevels, unsigned int);
     DECLARE_INPUT_EXCEPTION(ExcNoObserveElement,
             << "Failed to find the observe element with snap region: " << EI_RegionName::qval
-            << " close to the initial observe point. Using maximal number of neighbour levels: " << EI_NLevels::val << "\n");
+            << " close to the initial observe point. Change maximal distance of observe element." << "\n");
+
+    /// Helper enum specifies settings of update in point_projection method
+    enum ProjectionCases {
```

Comment:
Check if enum is used, possibly remove.
Check the same way and remove unused local variables in find_observe_point method.